cmd/kubeadm/app/cmd/config.go

```diff
@@ -380,7 +380,11 @@ func RunConfigView(out io.Writer, client clientset.Interface) error {
 		return err
 	}
 	// No need to append \n as that already exists in the ConfigMap
-	fmt.Fprintf(out, "%s", cfgConfigMap.Data[constants.InitConfigurationConfigMapKey])
+	if value, ok := cfgConfigMap.Data[constants.InitConfigurationConfigMapKey]; ok {
+		fmt.Fprintf(out, "%s", value)
+	} else {
+		fmt.Fprintf(out, "%s", cfgConfigMap.Data[constants.ClusterConfigurationConfigMapKey])
```

Comment:
Can we make sure that we can always get the data of `ClusterConfigurationConfigMapKey`? i.e., is it possible that the configmap doesn't have the data of such key?

Reply:
Seems like it's impossible :)